.npmrc

```diff
@@ -1,2 +1,5 @@
-# Set registry for @labkey scopes
 @labkey:registry=https://artifactory.labkey.com/artifactory/api/npm/libs-client/
+//artifactory.labkey.com/artifactory/api/npm/libs-client/:_password=QUtDcDVkS2lEWHF2WkF1TWdkOVREYlpnU2ZvMUdZNDZodWpHZzFEUGdjdEtNSFdzREFNRlJWRDFLRGlWdGZyUkx5NktBN0FqMQ==
+//artifactory.labkey.com/artifactory/api/npm/libs-client/:username=rosalinep
+//artifactory.labkey.com/artifactory/api/npm/libs-client/:email=rosalinep@labkey.com
+//artifactory.labkey.com/artifactory/api/npm/libs-client/:always-auth=true
```

Comment:
I assume this file was inadvertently checked in to this feature branch. Can you remove it?

Reply:
FYI, I reverted this change. As Susan messaged you about, you should change your api key.